lib/beacon_web/components/layouts.ex

```diff
@@ -119,6 +119,21 @@ defmodule BeaconWeb.Layouts do
     """
   end
 
+  def render_schema(%{__dynamic_page_id__: page_id, __site__: site} = assigns) do
+    %{raw_schema: raw_schema} =
+      site
+      |> Beacon.Loader.page_module_for_site(page_id)
+      |> Beacon.Loader.call_function_with_retry(:page_assigns, [page_id])
+
+    assigns = assign(assigns, :raw_schema, Jason.encode!(raw_schema))
+
+    ~H"""
+    <script type="application/ld+json">
+      <%= {:safe, @raw_schema} %>
```

Comment:
I'm planning to add a layer of validation in a future PR. Ideally we should validate against available types from https://schema.org